src/SearchResults/index.js

```diff
@@ -420,6 +420,8 @@ SearchResults.prototype.getFacetByName = function(name) {
     find(this.hierarchicalFacets, predicate);
 };
 
+var EMPTY_NORMALIZED_FACET_VALUES = (function() { var a = []; a.facetType = 'conjunctive'; return a; })();
```

Comment:
Actually, after reading the code, this property is only necessary for case it's a hierarchical facet. We may remove it :)